### PR TITLE
Stylesheet and rendering of grades.csv using tables

### PR DESCRIPTION
**Pull request by winstan**

### CODE REVIEW

**Comment by winstan** [comment]
@joelpittet - Could you spend 5-10 min on this just for sanity?

**Comment by joelpittet** [comment]
@winstan A review or what are you looking for?

**Comment by rxg** [comment]
Cool!  One alternative to using php to in-line the stylesheet is to use something like the following to refer to it in a distinct file:
`<link href="table.css" rel="stylesheet" type="text/css">`

This would replace `<style> ... </style>`

I'm not sure about the tradeoffs involved though.

File: table.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/* Stylesheet courtesy of https://github.com/mdn/learning-area/blob/master/html/tables/basic/minimal-table.css */
+html {
+  font-family: sans-serif;
+}
+
+</head>
```
Comment (joelpittet) [inline table.css]: Accidental tag

File: index.php [hunk 2]
```diff
@@ -145,13 +169,15 @@ function showGrades($ruser, $csvFile)
...
+<!DOCTYPE html>
 <html>
-<head>
+  <head>
     <meta charset="utf-8">
     <title>Handback</title>
-
-</head>
+    <style>
+    <?php include 'table.css'; ?>
```
Comment (joelpittet) [inline index.php]: This should be a `link` or inline like @rxg suggested.  I think for small amounts of CSS it's overkill to have a separate file. If it starts getting large and complicated it's good to keep them separate.

**Comment by winstan** [comment]
> Cool! One alternative to using php to in-line the stylesheet is to use something like the following to refer to it in a distinct file: `<link href="table.css" rel="stylesheet" type="text/css">`
> 
> This would replace `<style> ... </style>`
> 
> I'm not sure about the tradeoffs involved though.

Depends on if the style file is directly accessible by the webserver. In this case I think we should have it be accessible the same as the index.php is accessible... @natashamm

**Comment by winstan** [comment]
> @winstan A review or what are you looking for?

A review, yes; thanks.

File: index.php [hunk 3]
```diff
@@ -145,13 +169,15 @@ function showGrades($ruser, $csvFile)
     $htmlout .= "There is a problem with the handback dir... ";
 }
 ?>
-<!doctype html>
+<!DOCTYPE html>
```
Comment (joelpittet) [inline index.php]: Not needed change, both are acceptable.

**Review by joelpittet — CHANGES_REQUESTED** [review]
I'll poke at the PHP code in a few, just a quick glance

**Comment by rxg** [comment]
One other thought:  I realized that the code I wrote displays the table header even if you have no entries.  Not sure if this is a security issue (it's a little ugly I suppose).  If so let me know and I can suppress the table altogether if there are no entries.

File: index.php (hunk 2 above)
Comment (winstan) [reply index.php]: My thought is to actually have a 'default' css file that users modify. Now I'm not sure people will want to, but...
We don't want the average user making changes to code that is under version control.

Comment (winstan) [reply index.php]: So my vote is for link

**Pull request merged**